tagger/plot/bbbb.py

```diff
@@ -0,0 +1,288 @@
+import os, json
+from argparse import ArgumentParser
+
+from qkeras.utils import load_qmodel
+import awkward as ak
+import numpy as np
+import uproot
+import hist
+from hist import Hist
+
+#Plotting
```

Comment:
These should be defined in their own file so all our plotting scripts can call them and have the same style, again not essential for this PR